docs/SUMMARY.md

```diff
@@ -25,6 +25,7 @@
 
 * [Submitting proofs](3_guides/0_submitting_proofs.md)
 * [Build your first Aligned Application](3_guides/2_build_your_first_aligned_application.md)
+* [Modify ZkQuiz Questions](3_guides/2.2_modify_zkquiz_questions.md)
```

Comment:
It could be added as a subitem of Build your first Aligned Application

Reply:
Done!